### DISCUSSION TITLE
Workspace overview/map

### DISCUSSION THREAD

**Discussion by jdiez17** [YaLTeR/niri#759] [Ideas]
Hi all,

Thanks for making niri! It's a joy to use. But I keep getting "lost" when I have lots of horizontally and vertically stacked windows. It would be cool to have a utility that shows an overview "map" of the workspaces, for example when you press Mod+Tab.

Maybe something like this already exists and I don't know about it?

**Comment by YaLTeR** [comment]
Hey, it doesn't exist but there are ideas: #352
